### PR TITLE
Update project path for Leap15.3:WSL

### PR DESCRIPTION
**Pull request by mloviska**

### CODE REVIEW

**Comment by jlausuch** [comment]
LGTM!

**Review by jlausuch — APPROVED** [review]

File: xml/obs/openSUSE:Leap:15.3:WSL.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <openQA project_pattern="openSUSE:Leap:(?P&lt;version&gt;15.3):WSL"
-    dist_path="obspublish::openqa/openSUSE:Leap:15.3:WSL/standard/x86_64/wsl-appx">
+    dist_path="obspublish::openqa/openSUSE:Leap:15.3:WSL/images">
```
Comment (andrii-suse) [inline xml/obs/openSUSE:Leap:15.3:WSL.xml:2]: It probably should be `images/x86_64/kiwi-images-wsl/` : 
```
> rsync obspublish::openqa/openSUSE:Leap:15.3:WSL/images/*appx
rsync: link_stat "/openSUSE:Leap:15.3:WSL/images/*appx" (in openqa) failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1674) [Receiver=3.1.3]
> rsync obspublish::openqa/openSUSE:Leap:15.3:WSL/images/x86_64/kiwi-images-wsl/*appx
-rw-r--r--    182,353,539 2021/06/01 13:36:26 openSUSE-Leap-15.3-WSL.x86_64-153.1.1.0-Build1.1.appx
```

Comment (mloviska) [reply xml/obs/openSUSE:Leap:15.3:WSL.xml]: Updated, thanks!

**Review by andrii-suse — APPROVED** [review]

**Comment by andrii-suse** [comment]
It has been deployed:
```
> cd /opt/openqa-trigger-from-obs/
> git pull
remote: Enumerating objects: 21, done.
remote: Counting objects: 100% (21/21), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 11 (delta 7), reused 10 (delta 7), pack-reused 0
Unpacking objects: 100% (11/11), 1.10 KiB | 49.00 KiB/s, done.
From https://github.com/os-autoinst/openqa-trigger-from-obs
   e210abb..8e9530d  master     -> origin/master
Updating e210abb..8e9530d
Fast-forward
 t/obs/openSUSE:Leap:15.3:WSL/files_iso.lst          | 2 +-
 t/obs/openSUSE:Leap:15.3:WSL/print_openqa.before    | 8 ++++----
 t/obs/openSUSE:Leap:15.3:WSL/print_rsync_iso.before | 4 ++--
 xml/obs/openSUSE:Leap:15.3:WSL.xml                  | 2 +-
 4 files changed, 8 insertions(+), 8 deletions(-)
> make openSUSE:Leap:15.3:WSL
mkdir -p openSUSE:Leap:15.3:WSL && python3 script/scriptgen.py openSUSE:Leap:15.3:WSL
Generating scripts for openSUSE:Leap:15.3:WSL
OK
```

**Pull request merged**